test/compilable/testparse.d

```diff
@@ -179,3 +179,15 @@ void testIfConditionWithSTCandType()
 // https://issues.dlang.org/show_bug.cgi?id=20791
 extern(C++, "foo", )
 struct S {}
+
+/***************************************************/
+// https://issues.dlang.org/show_bug.cgi?id=22019
+void test22019()
+{
+    final switch (1)
+    {
+        case 1,:
+        case 2,3,:
+            break;
```

Comment:
Is there a test that verifies the error messages for `case ,:` and `case :`?

Reply:
I don't think so, though I did specifically avoid using `check(TOK.comma)` to avoid a bad error message.